carbon-design-system/carbon-addons-iot-react
# fix(table): increase min width for filterable column

Closes #3387 

**Summary**

- Increase min width for column from `62px` to `120px`
- Add text truncation for selected value in `ComboBox`

**Change List (commits, features, bugs, etc)**

- Change min width constant `MIN_COLUMN_WIDTH`
- Add css property for text truncation
- Update tests
- Update storyshot
- Update cypress image snapshot

**Acceptance Test (how to verify the PR)**

- Go to [this story](https://deploy-preview-3561--carbon-addons-iot-react.netlify.app/?path=/story/1-watson-iot-table-user-view-management--with-user-view-management)
- Choose any option in the `Select` column filter (combo box)
- Try to resize `Select` column
- Verify that value in the combo box field truncates
- Verify that min width for the column is `120px`

**Regression Test (how to make sure this PR doesn't break old functionality)**

- tests here



**Things to look for during review**
